chia/_tests/util/spend_sim.py

```diff
@@ -258,7 +258,7 @@ async def farm_block(self, puzzle_hash: bytes32 = bytes32(b"0" * 32)) -> tuple[l
         ]
         # Coin store gets updated
         generator_bundle: Optional[SpendBundle] = None
-        return_additions: list[Coin] = []
+        tx_additions = []
         return_removals: list[Coin] = []
```

Comment:
```suggestion
        additions = []
        removals = []
```

It looks like you could just call this `additions` and have it be assigned if there is a block.